subscriptions/reporting/subscription_reports.rst

```diff
@@ -0,0 +1,129 @@
+=================================
+Discover our subscription reports
+=================================
+
+As we now know well enough, understanding how our business is going and where it is going is the key
+to success. This is even more the case when we offer subscription services or products.
+
+Before getting to the heart of the matter, it is very important to recall certain concepts essential
+to the proper understanding of the following reports:
+
+- **Monthly Recurring Revenue (MRR)**: MRR is arguably the most important metric for subscription
+  businesses. Indeed, it shows the revenue earned with subscription-based products or services every
+  single month. It is a consistent number that you can use to track all your recurring revenue over
+  time, in monthly increments.
+
+- **Annual Run-Rate (ARR)**: ARR is the yearly version of MRR. Based on the current MRR, you can
+  estimate the coming year's performance with the Annual Run-Rate. However, this estimation does not
+  take variations and growth into account.
+
+.. image:: media/difference-between-MRR-and-ARR.png
+  :align: center
+  :alt: Difference between MRR and ARR in Odoo Subscriptions
+
+- **Non-Recurring Revenue (NRR)**: NRR shows the revenue earned for everything else than
+  subscription-based products or services. This includes gains of a rare or unique nature that are
+  unlikely to occur in the ordinary course of businesses.
+
+- **Customer Retention**: This concept consists of engaging existing customers to continue buying
+  products or services from your business. As you may know, customer retention is quite a challenge
+  because you must prove yourselves worthy of your customers' trust.
+
+- **Churn Rate**: Also known as the rate of attrition or customer churn, the churn rate can be
+  defined, in this case, as the percentage of service subscribers who discontinue their subscriptions
+  within a given time period. We can distinguish two types of Churn:
+
+  - **Logo Churn**: It corresponds to the subscription cancellation rate.
+
+  - **Revenue Churn**: It corresponds to the monthly recurring revenue loss rate.
+
+  .. note::
+     Let's imagine an increase of 2$ on a service subscription.
+
+     - We loss three customers out of the initial 20 which gives us a **Logo Churn** of 15%.
+     - | Therefore, the 56$ of MRR difference out of the initial 600$ gives us a **Revenue Churn**
+       | of 9,33%.
+
+     .. image:: media/difference-between-logo-churn-and-revenue-churn.png
+        :align: center
+        :alt: Difference between logo churn and revenue churn in Odoo Subscriptions
+
+     As a reminder, though they seem to evolve in the same direction most of the time, it might not
+     be the case all time.
```

Comment:
```suggestion
     Reminder: even though they seem to evolve in the same direction most of the time, it might not
     be the case all the time.
```